manifest/crds/sparkoperator.k8s.io_sparkapplications.yaml

```diff
@@ -2563,9 +2563,6 @@ spec:
               type: integer
             submissionID:
               type: string
-            terminationTime:
```

Comment:
Why this change?

Reply:
Didn't see your message above. Removing the check is fine for now before we can add `nullable: true`. I think we need to remove this check for all `metav1.Time` field in both `SparkApplication` and `ScheduledSparkApplication`.

Reply:
Actually a quick fix is to add `omitempty`.

Reply:
It seems that `controller-gen` supports the `+nullable` tag. Can we try that?

Reply:
Looks like `nullable` support is only available beginning 1.14. So I guess for now we need to remove this check. Actually I don't even think validation on `status` makes any sense for now. Let's remove validation for `status` all together.

Reply:
Yes, nullable support starts with 1.14.  Agreed, we can remove the validations for status. Let me do that

Reply:
I did remove the validations manually. I don't see any usable marker in https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/validation.go which would achieve the same ? Let me know if there is one.

Reply:
See https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/validation.go#L163.

Reply:
Not sure on https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/validation.go#L163 . I think this is the same `+nullable` tag which sets `nullable:true` in the CRD. It will let the status field be null but  if the `status` field is not null, it will still validate the types and we will run into the same `terminationTime` issue. Maybe I am missing something here.

Reply:
Yes, that's right. But for `status` as a whole, it's type is `object`, and I think they fixed the issue with empty `status`. We can still apply `+nullable` to those `metav1.Time` fields to solve the issue.

Reply:
`+nullable` will need 1.14+ support as we discussed before. I can do that in another PR if you think that's the right way.

Reply:
Yes, I meant for now we just remove validation for status, and in the future, we should add it back and use `+nullable` on those time fields.